### PR TITLE
[SOAR-8054] Servicenow - Cloud enabled | Changed connection input

### PR DESCRIPTION
**Pull request by igorski-r7**
### Description

Describe the proposed changes:

  - Cloud enabled | Changed connection input `URL` to `instance`

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [X] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [X] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [X] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [X] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [X] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [X] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [X] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [X] Work fully completed
- [X] Functional
  - [X] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [X] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [X] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [X] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [X] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

File: plugins/servicenow/help.md [hunk 1]
```diff
@@ -103,11 +103,7 @@ Example input:
 
 ```
 {
-  "create_data": {
-    "Description": "Bug report",
-    "ID": "58",
-    "date": "2021-08-20 18:12:00"
-  },
+  "create_data": "{\"Description\": \"Bug report\", \"ID\": \"58\", \"date\": \"2021-08-20 18:12:00\"}",
```
Comment (ablakley-r7) [inline plugins/servicenow/help.md:106]: Is this to pass validation? Example will be displayed as a string and not object.

File: plugins/servicenow/help.md [hunk 2]
```diff
@@ -154,7 +150,7 @@ Example input:
 
 ```
 {
-  "additional_fields": "{"description": "incident description"}",
+  "additional_fields": "{\"description\": \"incident description\"}",
```
Comment (ablakley-r7) [inline plugins/servicenow/help.md:153]: Same as above.

File: plugins/servicenow/help.md [hunk 3]
```diff
@@ -620,10 +616,7 @@ Example input:
 {
   "system_id": "9de5069c5afe602b2ea0a04b66beb2c0",
   "table": "catalog_category_request",
-  "update_data": {
-    "Description": "Bug report",
-    "ID": "58"
-  }
+  "update_data": "{\"Description\": \"Bug report\", \"ID\": \"58\"}"
```
Comment (ablakley-r7) [inline plugins/servicenow/help.md:619]: Same as above.

File: plugins/servicenow/help.md [hunk 4]
```diff
@@ -670,7 +663,7 @@ Example input:
 
 ```
 {
-  "additional_fields": "{"description": "incident description"}",
+  "additional_fields": "{\"description\": \"incident description\"}",
```
Comment (ablakley-r7) [inline plugins/servicenow/help.md:666]: Same as above.

**Review by ablakley-r7 — CHANGES_REQUESTED** [review]

**Review by ablakley-r7 — APPROVED** [review]

**Review by llaszuk-r7 — APPROVED** [review]

**Pull request merged**